minor editorial nits in Section 8 (Address Validation) (#4128)
quicwg/base-drafts: draft-ietf-quic-transport.md

An endpoint MAY include other frames with the PATH_CHALLENGE and PATH_RESPONSE
frames used for path validation.  In particular, an endpoint can include PADDING
frames with a PATH_CHALLENGE frame for Path Maximum Transfer Unit (PMTU)
discovery (see {{pmtud}}); it can also include its own PATH_CHALLENGE frame with
a PATH_RESPONSE frame.

An endpoint uses a new connection ID for probes sent from a new local address;

> see {{migration-linkability}}.  When probing a new path, an endpoint needs to

Reviewer: ```suggestion
see {{migration-linkability}}.  When probing a new path, an endpoint can
```

An endpoint MAY include PATH_CHALLENGE and PATH_RESPONSE frames that are used
for path validation with other frames.  In particular, an endpoint can pad a
packet carrying a PATH_CHALLENGE for Path Maximum Transfer Unit (PMTU)
discovery (see {{pmtud}}), or an endpoint can include its own PATH_CHALLENGE
with a PATH_RESPONSE.

When probing a new path, an endpoint might want to ensure that its peer has an
unused connection ID available for responses. The endpoint can send
NEW_CONNECTION_ID and PATH_CHALLENGE frames in the same packet, if the peer's

> active_connection_id_limit allow so. This ensures that an unused connection ID

Reviewer 2: ```suggestion
active_connection_id_limit permits. This ensures that an unused connection ID
```
